aspnetcore/security/authentication/azure-ad-b2c-webapi.md

```diff
@@ -170,12 +170,14 @@ To make an authenticated request to the web API, a bearer token is required. Pos
    |      <strong>Token Name</strong>       |                                  <em>&lt;token name&gt;</em>                                  |                                                                                                                   Enter a descriptive name for the token.                                                                                                                    |
    |      <strong>Grant Type</strong>       |                                           Implicit                                            |                                                                                                                                                                                                                                                                              |
    |     <strong>Callback URL</strong>      |                               `https://getpostman.com/postman`                                |                                                                                                                                                                                                                                                                              |
-   |       <strong>Auth URL</strong>        | `https://login.microsoftonline.com/tfp/<tenant domain name>/B2C_1_SiUpIn/oauth2/v2.0/authorize` |                                                                                                  Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name.                                                                                                  |
+   |       <strong>Auth URL</strong>        | `https://login.microsoftonline.com/<tenant domain name>/oauth2/v2.0/authorize?p=B2C_1_SiUpIn` |  Replace <em>&lt;tenant domain name&gt;</em> with the tenant's domain name. **IMPORTANT**: This URL must have the same domain name as what is found in `AzureAdB2C.Instance` in the web API's *appsettings.json* file. See call-out below for more information.            |
```

Comment:
> "See call-out below"

How about a cool dagger here? I love a dagger for this type of post-table reference.

> \*\*IMPORTANT\*\*: This URL must have the same domain name as what is found in \`AzureAdB2C.Instance\` in the web API's \*appsettings.json\* file. See NOTE\&dagger;.

... and on the note ...

> \> \&dagger;The **Auth URL** can be ...

Reply:
I'm down for a cool dagger!  

Enjoy your vacation!  Well-deserved!